python/node2vec_online.py

```diff
@@ -0,0 +1,248 @@
+import multiprocessing
+import time
+import mgp
+
+from itertools import chain, repeat
+from inspect import cleandoc
+
+from mage.node2vec_online_module.w2v_learners import (GensimWord2Vec)
+from mage.node2vec_online_module.walk_sampling import (StreamWalkUpdater)
+
+
+class Node2VecContext:
+    def __init__(self):
+        self._updater = None
+        self._learner = None
+        self._start_time = None
+
+    @property
+    def updater(self):
+        return self._updater
+
+    @property
+    def learner(self):
+        return self._learner
+
+    @property
+    def start_time(self):
+        return self._start_time
+
+    @updater.setter
+    def updater(self, updater):
+        self._updater = updater
+
+    @learner.setter
+    def learner(self, learner):
+        self._learner = learner
+
+    @start_time.setter
+    def start_time(self, start_time):
+        self._start_time = start_time
+
+    def is_initialized(self):
+        return (
+                self._learner is not None
+                and self._updater is not None
+                and self._start_time is not None
+        )
```

Comment:
Is mode uninitialized if any of these are None?

Reply:
No, you are right, just updater and learner.